cmd/kind/create/cluster/createcluster.go

```diff
@@ -50,15 +52,25 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "cluster context name")
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
+	cmd.Flags().Int32Var(&flags.ControlPlanes, "control-planes", 1, "number of control-plane nodes in the cluster")
+	cmd.Flags().Int32Var(&flags.Workers, "workers", 0, "number of worker nodes in the cluster")
 	cmd.Flags().StringVar(&flags.ImageName, "image", "", "node docker image to use for booting the cluster")
 	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "retain nodes for debugging when cluster creation fails")
 	cmd.Flags().DurationVar(&flags.Wait, "wait", time.Duration(0), "Wait for control plane node to be ready (default 0s)")
 	return cmd
 }
 
 func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
+	if cmd.Flags().Changed("config") && (cmd.Flags().Changed("control-planes") || cmd.Flags().Changed("workers")) {
```

Comment:
`control-planes` here too. Could the flag name be moved into a const if we are referencing it so many times?